### PR TITLE
Mobile/articles

### PR DESCRIPTION
**Pull request by BENTOTV**

### CODE REVIEW

**Review by widhofaisal — COMMENTED** [review]
mantap mas, cuman perlu dirapihin aja response nya. filed yang ga diperluin mending di hapus aja

File: controller/user/articles.go [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	}
+
+	//Populate Pictures field for each article
+	for i := 0; i < len(articles); i++ {
+		config.DB.Model(&articles[i]).Association("Pictures").Find(&articles[i].Pictures)
+	}
+
+	return c.JSON(http.StatusOK, map[string]interface{}{
+		"message": "success to retrieve latest articles data",
+		"data":    articles,
```
Comment (dvnggak) [inline controller/user/articles.go:35]: for improvement maybe can use custom response for excluding unimportant field 👍🏻

File: controller/user/articles.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	like.User_id = user_id
+	if err := config.DB.Save(&like).Error; err != nil {
+		log.Print(color.RedString(err.Error()))
+		return c.JSON(http.StatusBadRequest, map[string]interface{}{
+			"status":  400,
+			"message": "bad request",
+		})
+	}
+
+	// // Populate Pictures field for each product
```
Comment (dvnggak) [inline controller/user/articles.go:143]: better to keep the code clean from unused code / comment

**Review by dvnggak — APPROVED** [review]

**Review by mnaufalrifqir — COMMENTED** [review]
mantap

**Pull request merged**